src/Debugger/Main.elm

```diff
@@ -234,6 +235,22 @@ wrapUpdate update msg model =
       , Cmd.none
       )
 
+    Pause ->
+      case model.state of
+          Running _ ->
+              let
+                size = History.size model.history
+              in
+              if size == 0 then
+                ( model, Cmd.none )
+              else
+                ( jumpUpdate update (size - 1) model
```

Comment:
This does the same thing as dragging the slider.

If there is no history, `jumpUpdate` gets stuck in an infinite loop, so we explicitly check if there’s any history and do nothing otherwise.